packages/stable/src/types.ts

```diff
@@ -209,9 +209,17 @@ export interface ApiKeyRequest {
   serviceAccountId: CogniteInternalId;
 }
 
+export type ArrayPatchString =
+  | { set: Set<string> }
+  | { add?: Set<string>; remove?: Set<string> };
```

Comment:
Shouldn't it be just an array here?

Reply:
Nope. They need to be unique.

Reply:
have you checked that this actually works?
```
JSON.stringify(new Set("a", "b"))
```
prints  `{}`

Reply:
tbh, that's up to the api to do such validations.
sets are not necessarily easy to work with

Reply:
That seems like an unfortunate behaviour.